app/views/downloads/download_linux.html.erb

```diff
@@ -12,49 +12,83 @@
 
   <h3>Debian/Ubuntu</h3>
   <p>For the latest stable version for your release of Debian/Ubuntu</p>
-  <code># apt-get install git</code>
+  <pre># apt-get install git</pre>
   <p>For Ubuntu, this PPA provides the latest stable upstream Git version</p>
-  <code># add-apt-repository ppa:git-core/ppa</code>
-  <code># apt update; apt install git</code>
+  <pre># add-apt-repository ppa:git-core/ppa</pre>
```

Comment:
These pre blocks seem to have much more vertical whitespace, which makes the list much longer. I think we'd need some CSS to go with this.

Reply:
I will make a commit to do that.

Reply:
Here is the commit: 3a6af8947cd77f377a534ae7e9e7f98f0e7778aa